source/partials/application/_readme.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if application.private_repo? %>
+  <span class="govuk-error-message">
```

Comment:
Could do - though perhaps a bit confusing to use a form error on a page which has no form or user input :) I'd swap it out if we were rolling out our own `govuk-error-message` class, but this is built into the tech docs gem, so it wouldn't give us an opportunity to delete any code. Personally think it's fine as-is - what do you think?

Reply:
That makes sense to me. The mitigation risk is that if the design system ever change their error messages for any reason then this would be an extra source of truth, although I imagine the design system team would project breaking changes like this very loudly anyway. Feel free to resolve.